Resources/Prototypes/_NF/Roles/Jobs/Wildcards/pilot.yml

```diff
@@ -0,0 +1,38 @@
+- type: job
+  id: Pilot
+  name: job-name-pilot
+  description: job-description-pilot
+  playTimeTracker: JobPilot
+  requirements:
+    - !type:OverallPlaytimeRequirement
+      time: 120
+  startingGear: PilotGear
+  canBeAntag: true
+  icon: "JobIconPilot" 
+  supervisors: job-supervisors-captain
+  setPreference: true
+  access:
+  - Maintenance
+  - External
+  - Pilot
+
+- type: startingGear
+  id: PilotGear
+  equipment:
+    jumpsuit: ClothingUniformJumpsuitPilot
+    back: ClothingBackpackPilotFilled
+    shoes: ClothingShoesBootsPilot
+#    mask: ClothingMaskGasPilot # Doesn't exist yet
+    neck: ClothingNeckScarfPilot
+    eyes: ClothingEyesGlassesPilot
+    gloves: ClothingHandsGlovesPilot
+    head: ClothingHeadHatPilot
+    outerClothing: ClothingOuterCoatBomber
+    id: PilotPDA
+    ears: ClothingHeadsetAltPilot
+#    belt: ClothingBeltSalvageWebbing
+    pocket1: CrowbarRed
+    pocket2: Lighter
```

Comment:
Il honestly prefer to avoid giving misc items in job loadout and just the **JOB NEEDED ITEMS** as upstream has a Loudout PR and when its ready players should have the ability to have items like those on loadout.

Tho if the Loadout PR is merged on Frontier, i think we will mostly remove things like that from all job loadout.

Reply:
Can't think of any job specific items for a pilot. Dude spawns on a shop, no guarantee that there are any spare crowbars. Sure you can open airlock without one, but it takes forever.

Reply:
Im speaking about the lighter mostly.

Reply:
Removed it